AdzerkSDK/AdzerkSDK/AdzerkSDK.swift

```diff
@@ -347,7 +385,7 @@ public typealias ADZUserDBUserResponseCallback = (ADZUser?, Error?) -> ()
             "userKey": userKey,
             "interest": interest
         ]
-        pixelRequest(networkId, action: "optout", params: params, callback: callback)
+        pixelRequest(networkId, action: "interest", params: params, callback: callback)
```

Comment:
@subdigital Looks like this was a bug fix?  Good catch!  Does this mean that previous to this change setting interests on UserDB wouldn't work?

Reply:
Yep, that's correct. I had a "how did this ever work?" moment...